__test__/api-airline-delete.test.js

```diff
@@ -0,0 +1,42 @@
+import {
+  request,
+  describe,
+  test,
+  expect, //supertes
+  connectToDatabase, // couchbase
+  app, // REST application
+} from './imports'
+
+afterAll(async () => {
+  const { cluster } = await connectToDatabase()
+  await cluster.close()
+})
+
+describe('DELETE /api/v1/airline/{id}', () => {
```

Comment:
Shouldn't there be a test that deletes a non-existing document?

Reply:
Added 404 tests which deletes invalid doc id